closed/autoconf/custom-spec.gmk.in

```diff
@@ -34,6 +34,9 @@ endef
 OPENJ9_TOPDIR           := @OPENJ9_TOPDIR@
 OPENJ9OMR_TOPDIR        := @OPENJ9OMR_TOPDIR@
 
+J9JDK_EXT_VERSION       := 9.0.0.1
```

Comment:
Irina used 9.0.0.4 as the release version, not sure why.

Reply:
@irinarada ^

Reply:
oh!